prompt_toolkit/contrib/shortcuts.py

```diff
@@ -190,7 +190,7 @@ def get_input(message='',
 
     # Read input and return it.
     on_abort = AbortAction.RAISE_EXCEPTION if raise_exception_on_abort else AbortAction.RETURN_NONE
-    document = cli.read_input(on_abort=on_abort, on_exit=AbortAction.IGNORE)
+    document = cli.read_input(on_abort=on_abort, on_exit=AbortAction.RETURN_NONE)
```

Comment:
I think is interesting to have some compatibility with the raw_input command to easy replace readline with prompt_toolkit, but you wont have this raising a different exception. If you raise KeyboardInterrput and EOFError anyone can change raw_input with get_input and the rest of the code must works.

I like the raise_exception_on_abort param, but with your idea of compatibility you have to change the default value to True, and (in my opinion) include another parameter for raise_exception_on_exit with default value to True. When the value is False, i think the most interesting option is RETURN_NONE.